plugin/rpcplugin/process_windows.go

```diff
@@ -2,16 +2,647 @@ package rpcplugin
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"syscall"
+	"unicode/utf16"
+	"unsafe"
+
+	pkgerrors "github.com/pkg/errors"
 )
 
+type process struct {
+	command *cmd
+}
+
 func newProcess(ctx context.Context, path string) (Process, io.ReadWriteCloser, error) {
-	// TODO
-	return nil, nil, fmt.Errorf("not yet supported")
+	ipc, childFiles, err := NewIPC()
+	if err != nil {
+		return nil, nil, err
+	}
+	defer childFiles[0].Close()
+	defer childFiles[1].Close()
+
+	cmd := commandContext(ctx, path)
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	cmd.ExtraFiles = childFiles
+	cmd.Env = append(os.Environ(),
+		fmt.Sprintf("MM_IPC_FD0=%v", childFiles[0].Fd()),
+		fmt.Sprintf("MM_IPC_FD1=%v", childFiles[1].Fd()),
+	)
+	err = cmd.Start()
+	if err != nil {
+		ipc.Close()
+		return nil, nil, err
+	}
+
+	return &process{
+		command: cmd,
+	}, ipc, nil
+}
+
+func (p *process) Wait() error {
+	return p.command.Wait()
+}
+
+func inheritedProcessIPC() (io.ReadWriteCloser, error) {
+	fd0, err := strconv.ParseUint(os.Getenv("MM_IPC_FD0"), 0, 64)
+	if err != nil {
+		return nil, pkgerrors.Wrapf(err, "unable to get ipc file descriptor 0")
+	}
+	fd1, err := strconv.ParseUint(os.Getenv("MM_IPC_FD1"), 0, 64)
+	if err != nil {
+		return nil, pkgerrors.Wrapf(err, "unable to get ipc file descriptor 1")
+	}
+	return InheritedIPC(uintptr(fd0), uintptr(fd1))
+}
+
+// XXX: EVERYTHING BELOW THIS IS COPIED / PASTED STANDARD LIBRARY CODE!
+//      IT CAN BE DELETED IF / WHEN THIS ISSUE IS RESOLVED: https://github.com/golang/go/issues/21085
```

Comment:
Review the bits above this comment. Ignore everything below except the other lines I'm adding a PR comment to.